### DISCUSSION TITLE
Let's find another name for `fc.tuple`

### DISCUSSION THREAD

**Discussion by dubzzz** [dubzzz/fast-check#1986] [Announcements]
Same idea as https://github.com/dubzzz/fast-check/discussions/1985 but this time regarding `fc.tuple`.

As for `Set`, if the [TC39 proposal concerning Record/Tuple](https://github.com/tc39/proposal-record-tuple) makes it, `fc.tuple` might be misleading.

Here are some ideas I got for this one:
- `fc.fixedArray`
- `fc.sizedArray`
- _your ideas are more than welcomed_

**Comment by dubzzz** [comment]
Possible ideas here: https://github.com/tc39/proposal-record-tuple/issues/82#issuecomment-733853953

**Comment by nth-commit** [comment]
You could consider naming it based on the operation, rather than the type of the generated value: `zip`.

Similar to how "zipping" two lists often means combines the elements sequence-wise. "zipping" two generators combines the values of those generators sequence-wise. I know it's not really "sequence-wise" in the latter case but there seems to be some intuitive similarity there, perhaps based on the functor-ness of a list and an `fc.Arbitrary`.

**Comment by dubzzz** [comment]
TC39 proposal being partially dropped let's close the discussion for now.
